Entities/MOSRotating.cpp

```diff
@@ -1520,10 +1520,6 @@ void MOSRotating::PostTravel()
 // Description:     Updates this MOSRotating. Supposed to be done every frame.
 
 void MOSRotating::Update() {
-#ifndef RELEASE_BUILD
```

Comment:
I believe this is a false assert. MOID is used nowhere in Assert(), it should only be important for travel.

This was being triggered with held HDFIrearms that hadn't been added to movableman having their update called (through an actor's attachables heirarchy) - but that shouldn't be an issue.

Reply:
Hmmm, were you having this issue in the dev branch, or in your large map optimizations?
You're right that it should be mostly safe to get rid of this (though I guess lua stuff that checks moid can have problems, but lua people shouldn't use moids anyway). I'm just curious.

Reply:
The issue was spotted in my optimisation branch, but I’m fairly confident it’s in dev too, as my optimisations didn’t change how MOIDs were assigned, just the collision detection side. This assert triggers regardless of collision detection.

I spotted this while crash-sweeping with very high actor counts, so I suspect it’s just a rare trigger.